packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryResetPassword/en.ftl

```diff
@@ -5,3 +5,9 @@ create-new-password-header = Create new password
 account-restored-success-message = You have successfully restored your account using your account recovery key. Create a new password to secure your data, and store it in a safe location.
 # Feedback displayed in alert bar when password reset is successful
 account-recovery-reset-password-success-alert = Password set
+# An error case was hit that we cannot account for.
+account-recovery-reset-password-unexpected-error = Unexpected Error Encountered
```

Comment:
Is there a reason this is in title case (all three words capitalized). If it's a title could you indicate that in the ID name (e.g. `account-recovery-reset-password-unexpected-error-title`)  or a comment?

Reply:
Nope just a message. I've changed it.